### ISSUE TITLE
Incorrect example on crates.io web page

### ISSUE THREAD

**Issue by ChocolateLoverRaj** [nukeop/is-odd#7]
This is what https://crates.io/crates/is-odd shows:
```rs
extern crate is_odd;
use is_odd::IsOdd;

let _i : i32 = 1;
println!("{}", _i.is_odd()); // prints false
```
It says "prints false" when in this GitHub repo's README it says "prints true". I'm not sure why https://crates.io/crates/is-odd doesn't match this repo but the crates.io README should be updated. I actually thought this library doesn't work properly from looking at crates.io.

**Comment by nukeop** [comment]
Yes, I fixed it in the readme 4 years ago, but never got around to fixing it in the deployed crate.
